src/main/java/net/dv8tion/jda/api/entities/Activity.java

```diff
@@ -241,7 +251,7 @@ static Activity of(@Nonnull ActivityType type, @Nonnull String name, @Nullable S
      */
     static boolean isValidStreamingUrl(@Nullable String url)
     {
-        return url != null && url.matches("https?://(www\\.)?twitch\\.tv/.+");
+        return url != null && url.matches("https?://(www\\.)?(twitch\\.tv/|youtube.com/watch\\?v=).+");
```

Comment:
Does this really work?
Testing it in Regex101 gives [some errors](https://regex101.com/r/XmfbR0/1) about the / being unescaped.

Reply:
Which language did you have Regex101 set to? Not all languages implement regex the same way.

Reply:
I did have it on the default one which in this case was PHP.
Javascript does also return the same issue while Golang and Python don't.
It's a bit sad that Regex101 doesn't have the syntax/system for Java, but I guess Python would be close enough in that case.

Reply:
It was just pointed out in the Discord that the url of youtube is missing a `\\` before a dot.

Reply:
you can't just copy java -> regex101, cuz in java, every \ is double

Reply:
Also wanted to point that out in the commit right now. Since `.tv` was also escaped, I'm pretty sure it also has to be escaped. The slash shouldn't matter since it's not a special char like a dot, a question mark, etc.

Reply:
This is what you get when you fix the regex for non-java regex engines: https://regex101.com/r/XL5Rel/1 and it looks 100% ok to me

Reply:
The dot after youtube is still not escaped, making it a quantifier according to regex101

Reply:
Ok, that one i didn't see. Thanks